### ISSUE TITLE
StoreAsText default value

### ISSUE THREAD

**Issue by pyscripter** [EtheaDev/SVGIconImageList#47]
**UDataModule.dfm saved as text**
StoreAsText False: UDataModule.dfm size 512 Kb
StoreAsText True: UDataModule.dfm size 314 Kb

**UDataModule.dfm saved as binary**
StoreAsText False: UDataModule.dfm size 228Kb
StoreAsText True: UDataModule.dfm size 230Kb

I haven't done any benchmarks but I would bet StoreAsText True loads faster especially when the form is saved as Text.  (in that case there is an extra conversion HexToBin).

So I am not sure what the benefit of StoreAsText is (extra complexity) and it appears to me that True should be the default value of StoreAsText.

**Comment by carloBarazzetta** [comment]
I agree that StoreText should be the default mode to store data into dfm, and is the best way for source control so you can see the differences made to the specific svg code of the icons...
In view of the release of 2.0 we can establish some changes that are not backwards compatible.
